webapp/src/main/java/org/apache/atlas/web/rest/EntityREST.java

```diff
@@ -97,12 +98,14 @@ public class EntityREST {
     private final AtlasTypeRegistry      typeRegistry;
     private final AtlasEntityStore       entitiesStore;
     private final ESBasedAuditRepository  esBasedAuditRepository;
+    private final AtlasRelationshipIndexerService atlasESIndexService;
 
     @Inject
-    public EntityREST(AtlasTypeRegistry typeRegistry, AtlasEntityStore entitiesStore, ESBasedAuditRepository  esBasedAuditRepository) {
+    public EntityREST(AtlasTypeRegistry typeRegistry, AtlasEntityStore entitiesStore, ESBasedAuditRepository  esBasedAuditRepository, AtlasRelationshipIndexerService atlasESIndexService) {
```

Comment:
- Why we are injecting implementation ? and not the interface for `AtlasRelationshipIndexerService` ?